tvm-tac/src/main/kotlin/org/ton/tac/Stack.kt

```diff
@@ -510,74 +533,290 @@ class Stack(
     data class TacInstInfo(
         val inputs: List<Pair<String, TacStackValue>>, // input + input name from spec
         val outputs: List<TacStackValue>,
-        val continuationMap: Map<String, Int>,
+        val continuationMap: Map<String, ContinuationId>,
+        val metaObjects: MutableList<Pair<String, TacStackValue>>,
```

Comment:
What do you mean by `metaObjects`? Are these instruction's operands? (Parameters, that are coded right in instruction)